Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -1186,9 +1214,67 @@ static _WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
     m_backgroundWebView = nil;
 }
 
+WebExtensionContext::EventListenterTypeCountedSet WebExtensionContext::loadBackgroundPageListenersFromStorage()
+{
+    NSDictionary *extensionState = currentState();
```

Comment:
I'd change this line to `NSData *listenersData = [currentState() objectForKey:backgroundPageListenersKey]` to shorten the next line.